docs/examples/datasets/pipelines.py

```diff
@@ -350,3 +350,91 @@ def test_multiple_pipelines(client):
     )
     assert df.shape == (4, 4)
     assert found.sum() == 4
+
+
+# docsnip aggregate_with_str_window
+@meta(owner="data-eng-oncall@fennel.ai")
+@dataset
+class ActivityGroupedDataset:
+    timestamp: datetime
+    user_id: int = field(key=True)
```

Comment:
Minor but could you please put key field at the top?

Reply:
okie